examples/two-column-layout/src/app.tsx

```diff
@@ -0,0 +1,36 @@
+/** @jsx h */
+import { autocomplete } from '@algolia/autocomplete-js';
+import { h, render } from 'preact';
+
+import { querySuggestionsPlugin } from './plugins/querySuggestionsPlugin';
+import { recentSearchesPlugin } from './plugins/recentSearchesPlugin';
+
+import '@algolia/autocomplete-theme-classic';
+
+autocomplete({
+  container: '#autocomplete',
+  placeholder: 'Search products, articles, and FAQs',
+  debug: true,
+  autoFocus: true,
+  openOnFocus: true,
+  plugins: [recentSearchesPlugin, querySuggestionsPlugin],
+  render({ elements }, root) {
+    const {
+      recentSearchesPlugin: recentSearches,
+      querySuggestionsPlugin: querySuggestions,
+    } = elements;
+
+    render(
+      <div className="aa-PanelLayout aa-Panel--scrollable">
+        <div className="aa-PanelSections">
```

Comment:
We could use another prefix starting from here, to make it clear that those styles are specific to this example and not part of the `@algolia/autocomplete-theme-classic` package. Wdyt?

Reply:
Indeed, great point, I saw in other examples that we're using the prefix `aa-`, for example [here](https://github.com/algolia/autocomplete/blob/next/examples/github-notification-filters/style.css#L43), but maybe we just want to remove it so the distinction is clear as you said

Any opinion on this @sarahdayan?

Reply:
Why not re-using as much as possible from `@algolia/autocomplete-theme-classic` @dhayab ?

Reply:
For now, we chose to follow what we did in other examples.